trakt/sync.py

```diff
@@ -212,6 +212,23 @@ def search_by_id(query, id_type='imdb'):
     yield results
 
 
+@post
+def checkin_media(media, app_version, app_date, message="", sharing={},
```

Comment:
the default value for `sharing` here should be `None` to prevent those values being cached in subsequent calls to this function.

Reply:
Thanks, done